tests/integ/scala/test_function_suite.py

```diff
@@ -1482,20 +1486,6 @@ def test_to_timestamp_all(to_type, expected, session, local_testing_mode):
                 ),
             ],
         ),
-        (
-            to_timestamp_tz,
```

Comment:
Why was this example removed?

Reply:
it duplicates the first case `to_timestamp_tz` in the parameterize